index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Document</title>
+</head>
+<body>
+    <main>
+
+    <form id="form">
+        <input type="text" id="input">
+        <button id="button">Submit</button>
```

Comment:
Este não é um simples `button`, é um button de submição de formulário por isso tem de ter a propriedade `type="submit"`
```suggestion
            <button type="submit" class="space iput">Criar</button>
```

https://www.w3schools.com/tags/tag_button.asp